src/test/java/org/folio/rest/api/TestBaseWithInventoryUtil.java

```diff
@@ -42,12 +44,12 @@
   protected static final String TEMPORARY_LOCATION_ID_KEY = "temporaryLocationId";
   protected static final String EFFECTIVE_LOCATION_ID_KEY = "effectiveLocationId";
 
-  protected static final String MAIN_LIBRARY_LOCATION = "Main Library";
-  protected static final String SECOND_FLOOR_LOCATION = "Second Floor";
-  protected static final String ANNEX_LIBRARY_LOCATION = "Annex Library";
-  protected static final String ONLINE_LOCATION = "Online";
-  protected static final String THIRD_FLOOR_LOCATION = "Third Floor";
-  protected static final String FOURTH_FLOOR_LOCATION = "Fourth Floor";
+  public static final String MAIN_LIBRARY_LOCATION = "Main Library";
+  public static final String SECOND_FLOOR_LOCATION = "Second Floor";
+  public static final String ANNEX_LIBRARY_LOCATION = "Annex Library";
+  public static final String ONLINE_LOCATION = "Online";
+  public static final String THIRD_FLOOR_LOCATION = "Third Floor";
+  public static final String FOURTH_FLOOR_LOCATION = "Fourth Floor";
```

Comment:
Why have locations needed to be made public for a call number related test?

Reply:
I've moved data provider classes to a separate package, and `src/test/java/org/folio/rest/api/testdata/ItemEffectiveLocationTestDataProvider.java` uses this properties.